manga_auto_translator/segmentation_Unet/predict.py

```diff
@@ -0,0 +1,89 @@
+# USAGE
+# python predict.py
+# import the necessary packages
+import copy
+
+from pyimagesearch import config
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import cv2
+import os
+
+def prepare_plot(origImage, origMask, predMask,name):
+	# initialize our figure
+	figure, ax = plt.subplots(nrows=1, ncols=3, figsize=(10, 10))
+	# plot the original image, its mask, and the predicted mask
+	cpuOrigImage = origImage
+	ax[0].imshow(cpuOrigImage)
+	ax[1].imshow(origMask)
+	ax[2].imshow(predMask)
+	# set the titles of the subplots
+	ax[0].set_title("Image")
+	ax[1].set_title("Original Mask")
+	ax[2].set_title("Predicted Mask")
+	# set the layout of the figure and display it
+	figure.tight_layout()
+	figure.show()
+	plt.savefig(config.BASE_OUTPUT+name)
+
+def make_predictions(model, imagePath):
```

Comment:
It may be better to create custom module to make queries on model, like ModelAPI class.